config/initializers/rack-attack.rb

```diff
@@ -39,15 +39,6 @@ def user_id
 
   ### Throttle Requests ###
 
-  # Throttle all requests by user id
```

Comment:
I'm not sure if we want to completely remove this even though we have caching, as it is possible that different resources are being requested that may result in some form of cache eviction eventually. I think it would still be safer to leave this in